### PR TITLE
Added 2 tools that was missing and changed owner of Tinfoil Security …

### PR DESCRIPTION
**Pull request by jishnurippon**
…(acquired by Synopsys) in OWASP's Vulnerability Scanning Tools list

### CODE REVIEW

**Comment by kingthorin** [comment]
Please address the conflict., and undo whatever formatting so that the changes are obvious.

**Comment by jishnurippon** [comment]
> Please address the conflict., and undo whatever formatting so that the changes are obvious.

Conflict resolved

File: _data/tools.json [hunk 1]
```diff
@@ -1304,7 +1322,7 @@
       "note": null,
       "type": "DAST"
    },
-  {
+   {
```
Comment (kingthorin) [inline _data/tools.json:1325]: Please revert this change.

**Review by kingthorin — COMMENTED** [review]
Just one minor thing, otherwise looks good.

File: _data/tools.json (hunk 1 above)
Comment (jishnurippon) [reply _data/tools.json]: Completed

**Review by kingthorin — APPROVED** [review]
Thanks
